### PR TITLE
Fix example in `readme`

### PR DESCRIPTION
**Pull request by VincentPikand**
## Description

The example in `README.md` did not work "out of the box", so I supplemented it such that the user can just copy and paste it into their code.

## Type of change:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,13 +24,28 @@ Construct a new Atlas SDK client, then use the various services on the client to
 access different parts of the Atlas API. For example:
 
 ```go
-import "go.mongodb.org/atlas-sdk/admin"
+ctx := context.Background()
```
Comment (wtrocki) [inline README.md:27]:
```suggestion
import "go.mongodb.org/atlas-sdk/admin"

func example(){
  ctx := context.Background()
```

File: README.md [hunk 2]
```diff
@@ -24,13 +24,28 @@ Construct a new Atlas SDK client, then use the various services on the client to
...
+sdk, err := admin.NewClient(admin.UseDigestAuth(apiKey, apiSecret))
+if err != nil {
+    log.Fatalf("Error when instantiating new client: %v", err)
+}
 projects, response, err := sdk.ProjectsApi.ListProjects(ctx).Execute()
+if err != nil {
+    log.Fatalf("Could not fetch projects: %v", err)
+}
+fmt.Printf("Response status: %v\n", response.Status)
+fmt.Printf("Projects: %+v\n", projects)
```
Comment (wtrocki) [inline README.md:41]:
```suggestion
```

In most the cases, printing will likely to print some ugly pointer values.

File: README.md [hunk 3]
```diff
@@ -24,13 +24,28 @@ Construct a new Atlas SDK client, then use the various services on the client to
...
+```
+
+Run with
+```terminal
+MONGODB_ATLAS_PUBLIC_KEY=<public_key> \
+MONGODB_ATLAS_PRIVATE_KEY=<private_key> \
+go run <go_project_name>
 ```
```
Comment (wtrocki) [inline README.md:49]:
```suggestion
```

We have examples to show end to end experience. README gives only a glimpse of what api looks like

File: README.md (hunk 1 above)
Comment (wtrocki) [reply README.md:27]: Import name is essential for us.

**Review by wtrocki — APPROVED** [review]
Small suggestions. Thank you for fixing issue

File: README.md (hunk 3 above)
Comment (VincentPikand) [reply README.md]: removed

File: README.md (hunk 2 above)
Comment (VincentPikand) [reply README.md]: I kept the printing of `response.Status` as it looked nice, but removed printing for `projects`. Considered doing a for loop to print them nicely, but in the end decided against it.

File: README.md (hunk 1 above)
Comment (VincentPikand) [reply README.md]: updated

**Pull request merged**